source/_references/10-operations.md

```diff
@@ -167,8 +167,12 @@ Converts an image to grayscale.
 ### Modulate
 
 Modulates an image for Brightness, Saturation and Hue. Use this to for example brighten up or darken an image or 
-remove its colors. The value is meant as percentage, with which the different values are multiplied. 
-100 leaves the property untouched, 0 removes all of it. 
+remove its colors. For brightness and saturation, the value is meant as percentage, with which the different values are multiplied. 
+100 leaves the property untouched, 0 removes all of it.
+For hue, the value means how much the color circle should be rotated, 0 means 180 degrees to the left, 100 no rotation at all and 
+200 is 180 degrees to the right. 0 and 200 produce the same result. 150 for example rotates it 90 degrees to the right.
+
+See also this site for [more details about modulation](http://www.imagemagick.org/Usage/color_mods/#modulate).
```

Comment:
can we limit the range of hue below to 200 then?

Reply:
Sure ;) done